### DISCUSSION TITLE
Injecting config.yaml into Kratos Kubernetes pod via Terraform

### DISCUSSION THREAD

**Discussion by InverMN** [ory/kratos#1394] [Q&A] [answered]
Hello, today I have discovered Ory and after reading Kratos documentation I would like to set it up via Minikube and Terraform. The problem is that I don't have a simple solution for replacing ~/.kratos.yaml with my own tinkered kratos_config.yaml file. I know it's a more Terraform-related question than Kratos itself.

**Comment by aeneasr** [comment, marked as answer]
I think the helm charts could help you here:

- [Source](https://github.com/ory/k8s)
- http://k8s.ory.sh/helm/

**Reply by InverMN** [reply]
Okay, that allows me to pass the config YAML file via Terminal command and via Terraform helm provider. Thanks for the help!
